lib/project_types/script/ui/error_handler.rb

```diff
@@ -22,6 +24,11 @@ def self.pretty_print_and_raise(e, failed_op: nil)
         display_and_raise(failed_op: failed_op, **messages)
       end
 
+      private_class_method def self.append_msg(full_msg, msg_to_append)
+        full_msg << " " unless /\s$/.match?(full_msg)
```

Comment:
For my own understanding, is the purpose of this to avoid inserting a space after the `configuration_definition_errors_cause` because it ends in a new line?